tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/addons.py

```diff
@@ -33,8 +44,3 @@ def tap_first_item(self):
 
     def toggle_addon_status(self):
```

Comment:
You could do a check before and after that the checked state of _state_toggle_locator has changed

I guess when bug 1202388 is done, you could use something like:
GaiaBinaryControl(self.marionette, self._first_item_locator).enable()

So perhaps not worth the effor to do this anymore.